libraries/AP_HAL_ChibiOS/hwdef/CUAV-7-Nano/defaults.parm

```diff
@@ -0,0 +1 @@
+INS_ACCEL_FILTER,10
```

Comment:
I would recommend adding CAN_P1_DRIVER=1 to enable at least one CAN port...even if the protocol is not defaulted, it allows a user to use MAVLINK CAN inspector in MP...otherwise they would look and see nothing for CAN